extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1587,6 +1587,15 @@ export class AmpStoryPage extends AMP.BaseElement {
       return;
     }
 
+    if (!AmpStoryPage.installedAttachments) {
+      AmpStoryPage.installedAttachments = true;
```

Comment:
I don't think this is necessary, the `installExtensionForDoc` call right after looks smart enough

```
    if (!extLoaders) {
      extLoaders = rootNode[LOADER_PROP] = map();
    }
    if (extLoaders[extensionId]) {
      return extLoaders[extensionId];
    }
```

Reply:
True! I saw an error before that said "extension is already installed" or similar (and was breaking everything), but it doesn't show up now when I remove this check. It must have been sth else that caused that error (that's now reverted).